conf/build.manifest

```diff
@@ -38,6 +38,9 @@ apt-repos:
   - url: http://apt.tn.ixsystems.com/apt-direct/angelfish/nightlies/debian-security/
     distribution: bullseye-security
     component: main
+  - url: http://deb.debian.org/debian-security/
+    distribution: bullseye-security
+    component: main
```

Comment:
Just to be sure, wont that bring in any additional packages from that repo?

Reply:
Yes that is true. Because we have defined a mirror but the apt preferences we have set supersede this definition. I am building a build locally to ensure everything is good :)